index.html

```diff
@@ -216,6 +218,57 @@ <h2>Skills</h2>
                     </div>
                 </div>
             </section>
+            <!------------------------------------Contact section------------------------------------>
+            <section class="contact-form">
+                <div class="contact-header">
+                    <h2>I'm always interested in hearing about new projects, so if you'd
+                        like to chat please get in touch</h2>
+                </div>
+                <div class="form-container">
+                    <form class="contact" action="https://formspree.io/f/xknaqerb" method="post">
+                        <input name="1name" type="text" placeholder="Full name" maxlength="30" required>
+                        <input name="1email" type="email" placeholder="Email address" required>
+                        <textarea name="1text" type="textarea" maxlength="500"
+                            placeholder="Hey I'd like ot connect to..." required></textarea>
+                        <button class="btn">Get in Touch</button>
+                    </form>
+                    <p class="email">alijanrahimi1990@gmail.com</p>
```

Comment:
- [ ] Keep doing the great job 🤙

Reply:
Hi Dear @iambenkis,
Thanks for your guidance, I made the required changes